README.md

```diff
@@ -166,6 +178,8 @@ The player controls are shown under several well-established circumstances. This
 
 The default player controls UX implementation includes 4 optional app-specific sidebar buttons. You can set any or all of these to use as you see fit. This was built to allow for app-specific video overlay customization in anticipation for up to 4 new behaviors. Because these 4 sidebar buttons are built right into the default controls UX, they will automatically fade in/out with the rest of the video controls. There is no need to handle any of that logic or attempt to synchronize to the animation timings.
 
+We also think that there should be more gestures that will help user interact with out player. One of them is a double tap gesture. By doing double tap on the emtpy space of the player, user will be able to change video gravity from aspect to aspect fill. This gesture won't have any conflicts with controls show/hide gesture. 
```

Comment:
+We also think that there should be more gestures that will help user interact with out player. One of them is a double tap gesture. By doing double tap on the emtpy space of the player, user will be able to change video gravity from aspect to aspect fill. This gesture won't have any conflicts with controls show/hide gesture. 

should be:

+We also think that there should be more gestures that will help users interact with our player. One of these is a double tap gesture. By doing double tap in the empty space of the player, the user will be able to change video gravity from aspect fit to aspect fill. This gesture won't have any conflicts with controls show/hide gesture.